subprojects/core/src/testFixtures/groovy/org/gradle/cache/internal/locklistener/NoOpFileLockContentionHandler.java

```diff
@@ -26,6 +26,7 @@ public int reservePort() {
         return -1;
     }
 
-    public void pingOwner(int port, long lockId, String displayName) {
+    public boolean maybePingOwner(int port, long lockId, String displayName, long timeElapsed) {
```

Comment:
I was thinking about that. But I didn't want to add another state that depends on time here. I think that adds more complexity.